README.md

```diff
@@ -1,3 +1,26 @@
-# Fullstack Authentication Example with Next.js and NextAuth.js
+# Court-Share
+
+
+
+## Installation
+
+1. Clone the repository.
+2. Install dependencies with `npm install`.
+3. Install docker `https://docs.docker.com/get-docker/`
+
+## Running App locally
+1. Create a `.env`, add `URL_PRISMA` with an string value of `postgresql://postgres:password@localhost:5432/court-share?schema=public`
+2. In the prisma/schema.prisma file, edit datasource db block to this:
+```
+datasource db {
+  provider   = "postgresql"
+  url        = env("URL_PRISMA") // uses connection pooling
+  extensions = [postgis]
+}
+```
```

Comment:
Instead of doing this, lets use a `.env` file that defines the existing database URLs (POSTGRES_PRISMA_URL, POSTGRES_URL_NON_POOLING) with the local database URL. This saves the effort from having to modify the schema file.